pypesto/ensemble/ensemble.py

```diff
@@ -327,8 +328,45 @@ def __init__(self,
             self.predictions = predictions
 
     @staticmethod
-    def from_sample(result: Result, chain_index: int = 0, **kwargs):
-        x_vectors = result.sample_result.trace_x[chain_index].T
+    def from_sample(
+            result: Result,
+            chain_index: int = 0,
+            remove_burn_in: bool = True,
+            chain_slice: slice = None,
+            **kwargs,
+    ):
+        """Construct an ensemble from a sample.
+
+        Parameters
+        ----------
+        result:
+            A pyPESTO result that contains a sample result.
+        chain_index:
+            The chain that parameter vectors will be taken from.
+        remove_burn_in:
+            Exclude parameter vectors from the ensemble if they are in the
+            "burn-in".
+        chain_slice:
+            Subset the chain with a slice. Any "burn-in" removal occurs first.
+
+        Returns
+        -------
+        The ensemble.
+        """
+        x_vectors = result.sample_result.trace_x[chain_index]
+        if remove_burn_in:
+            if result.sample_result.burn_in is None or chain_index != 0:
+                burn_in = geweke_test(
+                    result,
+                    chain_index=chain_index,
```

Comment:
same here, the `burn_in` index should be calculated respect to the chain with lowest temperature...
maybe first we should discuss whether is making sense to run predictions for higher temperatures at all...

Reply:
any comments here too @yannikschaelte ?
to my mind, even if one would wish to look into higher temperatures, still the burn in should be w.r.t. the lowest one, no?

Reply:
that is a valid point ...I would say if one really wants to predict w.r.t. a higher temperature, then the geweke index should also be computed w.r.t. this temperature (as a "stand-alone" chain), but maybe we can drop that support in general and only do things for T=1, for simplicity, until someone wants this feature?

Reply:
I think we should drop it atm, since for debugging actually one should look into the MCMC chains, not on the predictions...
predictions is kind of one of the last things one does... and even, only after your MCMC chain is fully converged (T=1)

idk, if you want to keep it, just throw a warning in addition?

Reply:
fine to remove for me